xAPI.md

```diff
@@ -2643,23 +2643,52 @@ the owner of that extension IRI explicitly states that a particular form of code
 
 ## 1.0 Requests
 
-### 1.1 Request Types
+xAPI tracking is done via HTTP Requests from the Activity Provider (client) to the LRS (server).  This 
+specification offers guidance in some aspects of this communication.  Where no guidance is offered, it is 
+recommended that those implementing xAPI use current industry best practices.
+
+### 1.1 Request Methods
+
+xAPI supports the use of PUT, POST, GET, DELETE, and HEAD.  Use of other methods is beyond the scope of xAPI.  Each of the following sections details the use of each method and how xAPI may differ slightly.  For full 
+implementation details, see each of the [Resources](#resources) below.
 
 #### 1.1.1 HTTP PUT
 
+The HTTP PUT method is used to either create or update.  The enclosed entity is stored by the corresponding  
+Resource.  If it is new, it is created.  If there is a conflict, it is updated.  xAPI has a major exception to this 
+in that Statements are not allowed to be updated and thus there are rules around using PUT to update them.  Also in 
+xAPI, a PUT can only add a single Statement at a time.
+
+A major difference between PUT and POST is that PUT is always idempotent, meaning if the method is executed more than once, the same effect occurs as if it were executed just one time.  POST may require additional information 
+to be idempotent. 
+
 #### 1.1.2 HTTP POST
 
+Like PUT, the enclosed entity is stored by the corresponding Resource.  However, POST methods to any of the 
```

Comment:
I recommend deleting the first sentence entirely, and replacing with something like "POST is used to modify resources, such as by adding to them. When you POST to the resource for all Statements, you add one or more new Statements. When you POST to a Document API location, if any preconditions are met, you merge the contents with what's already there."